packages/api/src/command/medical/patient/consolidated-delete.ts

```diff
@@ -28,23 +27,21 @@ export async function deleteConsolidated(params: DeleteConsolidatedParams): Prom
 
   const getResourcesToDelete = async () => {
     try {
-      const bundle = await getConsolidated({
+      const bundle = await getConsolidatedPatientData({
         patient,
+        documentIds: docIds,
         resources,
       });
-      const resourcesFromFHIRServer = bundle.bundle.entry;
+      const resourcesFromFHIRServer = bundle.entry;
```

Comment:
just confirming this was intentional?

Reply:
good catch, but yeah... this is due to calling `getConsolidatedPatientData` instead of `getConsolidated`, since the latter also does conversion and we're not interested in this for this endpoint